src/formatDate.js

```diff
@@ -51,6 +51,37 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dataArr = date.split(fromFormat[3]);
+  const updatedDate = [];
+  const updatedYear = (lengthDiff, year) => {
+    if (lengthDiff > 0) {
+      return year < 30 ? `20${year}` : `19${year}`;
+    } else if (lengthDiff < 0) {
+      return year.slice(2);
+    }
+
+    return year;
+  };
+
+  const yearData = (format) => {
```

Comment:
```suggestion
  const yearData = (format) => {
```
Function's name should be verb so fix it everywhere